modules/katamari/src/main/ts/ephox/katamari/api/Type.ts

```diff
@@ -1,10 +1,26 @@
+const getPrototypeOf = Object.getPrototypeOf;
+
+const hasProto = (v: Object, predicate: (v: Object) => boolean, name: string): boolean => {
+  if (predicate(v)) {
+    return true;
+  } else {
+    // String-based fallback time
```

Comment:
Can we please add a comment that the string based fallback is only needed for IE 11? That way we can remove it once we drop IE support.